userspace/falco/configuration.cpp

```diff
@@ -105,8 +106,28 @@ void falco_configuration::init(const std::string& conf_filename, const std::vect
 	load_yaml(conf_filename, config);
 }
 
+static driver_mode_type get_driver_mode(const std::string& input){
+	// Set driver mode if not already set.
+	const std::unordered_map<std::string, driver_mode_type> driver_mode_lut = {
+		{"kmod",driver_mode_type::KMOD},
+		{"bpf",driver_mode_type::BPF},
+		{"modern_bpf",driver_mode_type::MODERN_BPF},
```

Comment:
I'd call this one `modern-bpf`. 
That is how it is used today in rpm/deb packages scripts, and this is what i will use for https://github.com/falcosecurity/falcoctl/pull/343.
Also, the systemd unit is called `modern-bpf` and the CLI flag too.